src/lib/formatters.js

```diff
@@ -19,6 +20,8 @@ function getTime(date) {
 }
 
 export function formatDate(timestamp) {
+  const { t } = useI18n()
+
```

Comment:
So now this function is a hook. Better to rename it (e.g. useFormattedDate) and move to /hooks dir. Check if it does not break anything inside Vue components. Remember, it must be used as a hook, not a function.